Views/searches.view.lkml

```diff
@@ -68,7 +69,7 @@ view: searches {
   dimension: search_terms {
     description: "The search term(s) that were queried."
     type:  string
-    sql:  replace(substring(${raw_search_terms}, 3, length(${raw_search_terms})-4),'+',' ') ;;
+    sql:  substring(${raw_search_terms}, 3, length(${raw_search_terms})-4) ;;
```

Comment:
This looks like a change unrelated to deduplicating extended dimensions. Is this for work in a different ticket?

Reply:
Ive reverted the sql.